### PR TITLE
Add testcase for setting/getting a pointer to a long

### PR DESCRIPTION
**Pull request by zecke**

### CODE REVIEW

File: src/UnifiedFFI-Tests/FFIExternalStructureTests.class.st [hunk 1]
```diff
@@ -216,12 +216,17 @@ FFIExternalStructureTests >> testStructWithArray [
...
 
 	s1 := FFITestPointerStructure new.
 	self assert: s1 getHandle isExternalAddress not.
 	self assert: s1 pointer isNull.
 	self assert: s1 nestedPointer isNull.
+
+	"Test that setting a pointer gets us a pointer back."
+	pointer := ExternalAddress gcallocate: 8.
+	s1 longPointer: (ExternalData fromHandle: pointer).
+	self assert: s1 longPointer getHandle equals: pointer.
```
Comment (zecke) [inline src/UnifiedFFI-Tests/FFIExternalStructureTests.class.st:231]: This returns an ExternalData and the handle is a SmallInteger. It should be a pointer?

**Comment by Ducasse** [comment]
Ping Esteban

**Comment by Ducasse** [comment]
Ping esteban

**Comment by zecke** [comment]
Ping to avoid staleness

**Comment by zecke** [comment]
Dear Staley McBot. This is a thing.

**Comment by zecke** [comment]
IIRC the test failures expose a real bug.

**Comment by MarcusDenker** [comment]
Not stale

**Review by guillep — COMMENTED** [review]
Hi @zecke, I want to move this one forward.

A test is a test, so I'd vouch to make it in.

Is there anything else to consider?

**Comment by zecke** [comment]
The test is failing and exposes an issue in UFFI.

**Comment by pavel-krivanek** [comment]
@zecke merge conflict

**Comment by estebanlm** [comment]
I solved conflict and create a new PR
@zecke can you check is this ok?

**Pull request closed without merging**